titapapunne/MoreThanWords
# mtw 92: fix bugs

Steps to test PR: 
Login and cancel. See if any warning pops up.

Standard Checklist:

Resolves #92
